lizmap/modules/admin_api/lib/Utils.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LizmapApi;
+
+class Utils
+{
+    /**
+     * Extracts and returns the relative path from a given path.
+     *
+     * @param string $path The directory path to process.
+     * @return string The relative portion of the path, formatted with a trailing slash.
+     */
+    public static function getRelativePath(string $path): string
```

Comment:
This method returns the last part of a path with a trailing slash not a relative path from a given path.